src/wallet/tx_builder.rs

```diff
@@ -560,6 +561,12 @@ impl<'a, D: BatchDatabase, Cs: CoinSelectionAlgorithm<D>, Ctx: TxBuilderContext>
         self.params.current_height = Some(height);
         self
     }
+
+    /// Set whether or not the dust limit is checked.
```

Comment:
Maybe we could also explain here that if the dust is allowed you might end up with a transaction that is non standard